Kudu.Core/Jobs/TriggeredJobSchedule.cs

```diff
@@ -50,6 +52,8 @@ public void Reschedule(DateTime lastRun, Schedule schedule = null)
                 Logger.LogInformation("Next schedule expected in " + nextInterval);
             }
 
+            _analytics.JobEvent(TriggeredJob.Name.Fuzz(), $"Reschedule {nextInterval}", TriggeredJob.JobType, String.Empty);
```

Comment:
This is the place where we reschedule the next run based on cron expression.   This should help us debug.